docker-compose.override.yml

```diff
@@ -7,6 +7,18 @@ services:
       - '443:443'
     volumes:
       - ./conf/ssl:/ssl
+      - ./conf/nginx/conf.d/dev.conf:/etc/nginx/conf.d/oscar.template
+
+  backup:
+    build: .
+    image: countable/s3-backup-job:1.0
+    volumes:
+      - $HOME/.aws/credentials:/root/.aws/credentials
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      - BACKUP_BUCKET=countable/backups
+      - BACKUP_CMD='mysqldump -uroot -pliyi oscar_mcmaster > dump'
+    restart: always
```

Comment:
We will also need to move the OscarDocuments folder to the backup, and set the frequency to daily instead of hourly.

Reply:
@ferdinandpolpol can we include a bash script to do this?

Reply:
Do you mean a script to manually execute the backups?

Reply:
Instead of mysqldump, we need a script to do that and also zip up and copy the OscarDocuments folder to s3.

Reply:
@countable I added the OscarDocuments backup here https://github.com/ferdinandpolpol/s3-backup-jobs/commit/1c4060c9989e2f5a181a3d360c02c1ccfd1e7459 , we are using my fork for the backup image.